qt/scientific_interfaces/Indirect/IndirectFitPlotPresenter.h

```diff
@@ -9,6 +9,7 @@
 #include "DllConfig.h"
 
 #include "IndirectFitPlotModel.h"
+#include "MantidAPI/IFunction_fwd.h"
```

Comment:
What is this included for?

Reply:
Must have been left over, I'll remove it

Reply:
I also spotted it in IndirectFitPlotModel.h file.